src/corecel/sys/Environment.cc

```diff
@@ -64,6 +65,9 @@ std::string const& getenv(std::string const& key)
  * specifying whether the default was used. The insertion result can be useful
  * for providing a diagnostic message to the user.
  *
+ * As with the general \c Environment class, the already-set values override
+ * whatever variables are in the system.
```

Comment:
Can you clarify a bit (what is 'the system', how can the value be already set?, etc.).
```suggestion
 * As with the general \c Environment class, the already-set values has priority over
 * whatever variables are in the system.
```